uint64 pedestrianID
GPSSample gpsSample
